docs/specs/tripgo.swagger.yaml

```diff
@@ -2801,7 +2831,8 @@ definitions:
           $ref: '#/definitions/Location'
         cost:
           type: number
-          description: Cost of this segment in local currency
+          description: |
+             **Deprecated.** Cost of this segment in local currency
```

Comment:
I believe that some Swagger tools support a `deprecated` boolean. Please add one.

Reply:
I added it, but it seems that `deprecated` on properties is only possible in OAS 3, and ReDocs doesn't support it. So I left it in the description too. Did the same thing for `disclaimer`.